### PR TITLE
chore: pick some commits from PR

### PR DESCRIPTION
**Pull request by Jack-Works**

### CODE REVIEW

File: packages/dashboard/src/components/DashboardFrame/index.tsx [hunk 1]
```diff
@@ -100,10 +99,9 @@ const PageTitle = styled(Grid)(({ theme }) => ({
 }))
 
 const Containment = styled(Grid)(({ theme }) => ({
-    overflow: 'auto',
     contain: 'strict',
     [theme.breakpoints.down('lg')]: {
-        minHeight: '100vh',
+        minHeight: `calc(100vh - 64px)`,
```
Comment (septs) [inline packages/dashboard/src/components/DashboardFrame/index.tsx:104]: suggest, add note for magic number (`64px`)

**Review by septs — APPROVED** [review]

File: packages/dashboard/src/components/DashboardFrame/index.tsx (hunk 1 above)
Comment (nuanyang233) [reply packages/dashboard/src/components/DashboardFrame/index.tsx:104]: Well，I will add it in #3061

File: packages/dashboard/src/initialization/isolated_bridge.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
+import { Environment, UnboundedRegistry, WebExtensionMessage } from '@dimensiondev/holoflows-kit'
+import { AsyncCall } from 'async-call-rpc'
+import { serializer } from '@dimensiondev/maskbook-shared'
+import { PluginMessages, setPluginServices, setService } from '../API'
+
+class WebExtensionExternalChannel extends WebExtensionMessage<any> {
+    constructor(domain: string, id = 'jkoeaghipilijlahjplgbfiocjhldnap') {
```
Comment (guanbinrui) [inline packages/dashboard/src/initialization/isolated_bridge.ts:7]: Move to constant.

File: packages/dashboard/src/Dashboard.tsx [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { lazy, Suspense } from 'react'
+
+const Dashboard = lazy(() => import('./initialization/Dashboard'))
+export function IntergratedDashboard() {
+    return (
+        <Suspense fallback="">
```
Comment (guanbinrui) [inline packages/dashboard/src/Dashboard.tsx:6]: It's WIP?

**Review by guanbinrui — APPROVED** [review]

File: packages/dashboard/src/Dashboard.tsx (hunk 3 above)
Comment (Jack-Works) [reply packages/dashboard/src/Dashboard.tsx:6]: It's by design. This is used to defer the import module graph.

**Pull request merged**